src/jsd/actuator.cc

```diff
@@ -696,10 +700,15 @@ void fastcat::Actuator::EgdProcess()
   jsd_egd_process((jsd_t*)context_, slave_id_);
 }
 
+void fastcat::Actuator::EgdClearErrors()
+{
+  jsd_egd_clear_errors((jsd_t*)context_, slave_id_);
+}
+
 void fastcat::Actuator::EgdReset()
```

Comment:
It seems that this function is never used.

Reply:
It is `actuator.cc:530` 
https://github.com/nasa-jpl/fastcat/blob/81de32c9d635e2af37e1afc37fa5fca5a437c326/src/jsd/actuator.cc#L530